int32 status #(0 = inactive, 1 = succeded, -1 = failed, 42 = working)
string feedback
